src/Controller/MetadataExposeDisplayController.php

```diff
@@ -270,20 +270,19 @@ public function castViaTwig(
               'format_strawberryfield.iiif_settings'
             )->get('pub_server_url');
             $original_context = $context + $context_embargo;
-            // Allow other modules to provide extra Context!
-            // Call modules that implement the hook, and let them add items.
-            \Drupal::moduleHandler()->alter(
-              'format_strawberryfield_twigcontext', $context
-            );
-            // In case someone decided to wipe the original context?
-            // We bring it back!
-            $context = $context + $original_context;
-            $cacheabledata = [];
```

Comment:
@patdunlavey can you keep please  $cacheabledata = []; in line 281?

Reply:
I'm afraid I don't understand. Restoring it to a position before `$this->renderer->executeInRenderContext` will simply result in it being overwritten, and putting it after will clobber the result of `$this->renderer->executeInRenderContext`, and putting it inside the inline function does nothing that I can see.

Reply:
Nevermind. I've gone ahead and updated the issue branch with the change as requested (and as I understand it).